test/test_immersed_advection.jl

```diff
@@ -98,7 +98,7 @@ for arch in archs
     @testset "Immersed tracer reconstruction" begin
         @info "Running immersed tracer reconstruction tests..."
 
-        grid = RectilinearGrid(arch, size=(20, 20), extent=(20, 20), topology=(Bounded, Bounded, Flat))
+        grid = RectilinearGrid(arch, size=(20, 20), extent=(20, 20), halo = (7, 7), topology=(Bounded, Bounded, Flat))
```

Comment:
👁️

Reply:
yeah I didn't see I am going only up to buffer 5, hence 6 should work